locale/src/locale.cc

```diff
@@ -51,7 +51,11 @@ namespace leatherman { namespace locale {
 
     string translate(string const& s, string const& domain)
     {
-        return boost::locale::translate(s).str(get_locale("", domain));
+        try {
+            return boost::locale::translate(s).str(get_locale("", domain));
+        } catch (exception const&) {
+            return s;
```

Comment:
If we were going to do something with memoizing, I'd switch back to directly accessing g_locales, and just return the untranslated string if the locale hadn't been initialized yet. That ends up masking whether we tried to use translate before initializing locales vs locales being busted, but it would be a graceful way to fail.